driftbase/flask/flaskproxy.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+import flask
+from werkzeug.local import LocalProxy
+from drift.core.resources.redis import RedisCache
+from sqlalchemy.orm import Session
+
+class _flaskProxy:
+    """
+    Proxy which allows type hinting flask globals. Import instead of
+    flask.g to benefit  from type hints.
+    """
+
+    # Note: Some of these are actually 'LocalProxy[T]', but for improved type hinting we will hint the underlying type.
+    redis : RedisCache
+    db : Session
+
+    def __getattr__(self, key):
+        return getattr(flask.g, key)
+    def __setattr__(self, name: str, value: Any) -> None:
+        flask.g.__setattr__(name, value)
```

Comment:
Any particular reason for not going through setattr(flask.g, name, value) here?

Reply:
I would want to confirm how that interacts with the wrapping LocalProxy, but probably it's equivalent. Will check when I'm dealing with the rest of the review.